packages/pharos/src/components/image-card/pharos-image-card.ts

```diff
@@ -404,8 +418,6 @@ export class PharosImageCard extends ScopedRegistryMixin(FocusMixin(PharosElemen
           ?checked=${this._isSelected}
           ?disabled=${this.disabled}
           name="Select ${this.title}"
-          @mouseenter=${this._handleMouseEnterSelectable}
-          @mouseleave=${this._handleMouseLeaveSelectable}
```

Comment:
These mouseevents and the moving of the `${this._renderCheckbox()}` in the templates are the focus of the fix.

Reply:
I think these changes look really good. Did you have a chance to yalc the changes into Workspace? I think it's fine, but we may want to confirm that the folder cards still look the same.

Reply:
I checked it out in Chrome and looks good. Safari can be a little tricky/tedious with the cookie mapping to test out yalc'd over changes locally.